api/test/models/commit_test.rb

```diff
@@ -26,4 +26,12 @@ class CommitTest < ActiveSupport::TestCase
 
     assert(commit.valid?)
   end
+
+  test "serialize/deserialize parent hashes" do
+    commit = commits(:rails_4ad93b)
+    commit.save
+
+    commit = Commit.find(commit.id)
+    assert_equal("2fe1ca0f058c3e9ffd7fa7616e27c03aed8dba7a", commit.parent_hashes[0])
+  end
```

Comment:
I see what you tried to do, but I don't think it does what you think it does. 

`commits(:rails_4ad93b)` is already saved in the database as soon as the tests starts. If it couldn't serialize the array of parents, it would fail as soon as you type `bin/rails test`.  

---

If you really wanna make sure serialization work, you should create a brand new commit and assign it an array of hash. Then save it, and check if the serialization worked. 

TBH though, that's a native feature of rails so I guess we can just remove the test.

Reply:
done